cmd/hub/mcscontroller/main.go

```diff
@@ -129,16 +134,14 @@ func main() {
 	}
 }
 
-// getWatcedhNamespace returns the Namespace the operator should be watching for changes.
-// An empty value means the operator is running with cluster scope.
+// getWatchedNamespace returns the Namespace the operator should be watching for changes.
+// An empty value means the operator is running with cluster scope (currently unsupported!).
 // Could support multiple, comma separated values, by setting the controller NewCache
 // option with cache.MultiNamespacedCacheBuilder().
 func getWatchedNamespace() (string, error) {
-	const nsEnvVar = "NAMESPACE"
-
-	ns, found := os.LookupEnv(nsEnvVar)
+	ns, found := os.LookupEnv(ENV_NAMESPACE)
 	if !found {
-		return "", fmt.Errorf("environment variable %s should be set", nsEnvVar)
+		return "", fmt.Errorf("environment variable %s should be set", ENV_NAMESPACE)
```

Comment:
if env var is a must, whats the sense in accepting also cli switch?

Reply:
falls back to the env variable if not specified by command line. I expect this to be standard behavior